backend/app/email_templates/base.html

```diff
@@ -0,0 +1,45 @@
+<div class="header">
+  <img src="https://flathub.org/img/logo/flathub-logo-toolbar.svg" alt="Flathub"/>
+</div>
+
+<hr/>
+
+<h1>{{ email_subject }}</h1>
+
+{% block content %}
+{% endblock %}
+
+<hr/>
+
+{% if app_id %}
+  You are receiving this email because you are a maintainer of the app {% include "app_name_and_id.html" %} on Flathub.
+{% else %}
+  You are receiving this email because you have an account on Flathub.
+{% endif %}
+
```

Comment:
No way to opt out? Not sure if we would need that

Reply:
I'm planning to add that at some point. It'll need a backend for storing preferences and a frontend for editing them, but that shouldn't be too difficult.

I think there's some emails we don't want people to opt out of, like when a token is generated. It might also be a good idea to notify people if a notification category is disabled.

Reply:
I think we agreeded yesterday, that we won't let people opt out, as it's central funtionallity.